server/resource_server/models/verification.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+import "time"
+
+type EmailVerificationData struct {
+	ID               uint      `gorm:"primaryKey; autoIncrement; not null" json:"id"`
+	UserId           uint      `gorm:"column:user_id; unique; not null" json:"user_id"`
+	VerificationCode string    `gorm:"column:verification_code; not null" json:"verification_code"`
+	ExpiresAt        time.Time `gorm:"column:expires_at; not null" json:"expires_at"`
+}
```

Comment:
can we have sql DDL for this?

Reply:
lets keep this for now, after i merged the development setup branch we cann add the ddl definition